src/game/actions.ts

```diff
@@ -1,4 +1,4 @@
-import { Direction } from "./enums";
+import { Direction, Spell } from "./enums";
```

Comment:
How come this wasn't caught by our CI? I suggest that we start running linters in the pipeline too. What do you think? There are some decent linters here: https://palantir.github.io/tslint/

Reply:
Good point! My team will work on beefing up our tests and lint integration this week

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/9f9acb2f-f431-4f37-93a1-a20d44fcb5b5?message=2aaad613-6a83-4a27-95b7-b5f4f90d368c).

Reply:
@pwerry click me!